### ISSUE TITLE
Project Wingman (895870)

### ISSUE THREAD

**Issue by TacticalTux** [ValveSoftware/Proton#4437]
# Compatibility Report
- Name of the game with compatibility issues:  Project Wingman
- Steam AppID of the game: 895870

## System Information
- GPU: GTX 1070
- Driver/LLVM version: 455.45.01
- Kernel version: 5.9.12-97-tkg-upds
- Link to full system information report as [Gist](https://gist.github.com/TacticalTux/16df99e7f437c775e35d6d6fc5c4ac03): https://gist.github.com/TacticalTux/16df99e7f437c775e35d6d6fc5c4ac03
- Proton version: 5.13-2

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

## Symptoms 
Upon starting the game, a black screen appears until a message stating "Fatal Error!" appears on screen.

## Reproduction
Install and start game in standard, non VR mode.

[steam-895870.log](https://github.com/ValveSoftware/Proton/files/5647128/steam-895870.log)

**Comment by TacticalTux** [comment]
A solution to this has been found by installing d3dcompiler_47 into the prefix from winetricks (or protontricks)

VR, however, is not currently playable. Instead of crashing, it freezes at the starting screen.

[steam-895870.log](https://github.com/ValveSoftware/Proton/files/5647186/steam-895870.log)

**Comment by NeoTheFox** [comment]
VR works for me after applying the same fix. I'm on AMDGPU/Mesa, maybe that's what is different.
There is another bug I'm experiencing with this game specifically - my joystick, Defender Cobra M5, would not persist as a configured input device. I have to repeat the configuration every time I restart the game. I suspect it's the long name of its kernel module. 
The name as it appears to the system is `www.vkb-pro.com Â© Alex Oz 2013 Defender COBRA M5 USB Joystick`
Here is a [screenshot](https://imgur.com/Df87HcB)

**Comment by SecularSteve** [comment]
The Anniversary Update has it's own set of issues. Mainly the steeper hardware requirements. 

You can no longer leave the graphics at default to play it on the Steam Deck: 

https://www.youtube.com/watch?v=JD6Lc6jpAGM

**Comment by smokcat** [comment]
3 days ago I was able to play this game with no issues, and now (for no apparent reason) I get the above described black screen with error message box.

proton logs are filled with the following error: "**wine: error opening ftrace file: Permission denied.**"
other games in the same install location run perfectly fine on proton.

issue persists even on a fresh clean install of steam and arch (archinstall script), game works on the steamdeck tough...

also, it seems weird to me that it even tries to do something to **ftrace**, afaik it's related to kernel tracing and debugging, seems weird and sketchy to me why it wants that suddenly. And i dont think it wanted access to that before, because that would have most certainly failed as ftrace is owned by root.
so as far as i can see the culprit is it suddenly needing ftrace.

[steam-895870.log](https://github.com/user-attachments/files/20611336/steam-895870.log)

![Image](https://github.com/user-attachments/assets/531af8cd-1219-498f-a7fd-e43bbfa90318)

![Image](https://github.com/user-attachments/assets/090b975a-fe0b-4236-a0e9-5e27bc23f109)

**Comment by simifor** [comment]
@TheKessler0 the game is working as expected over here in every proton version between 9 and experimental bleeding edge. Have you tried other proton versions? if so, are they also affected?

**Comment by smokcat** [comment]
@simifor turns out it works with proton experimental, **BUT** infuriatingly the launch option ("PROTON_LOG=1 WINEDEBUG=+all %command%") to enable proton debug log causes the "ftrace crash" issue i had, reason why it crashes without debug options in hotfix (default proton version) is unknown.
that is why initially it seemed to not work in any proton version.

TL;DR:
it crashes reliably under proton hotfix now (this is new and did **NOT** happen 4 days ago)
debug launch options reliably cause a crash
game works as expected in proton experimental

**Comment by ashtonx** [comment]
Still an issue, fix is useless atm. Tried hotfix and ge.

```
➜ protontricks 895870 d3dcompiler_47
pressure-vessel-wrap[39001]: W: Not sharing path --filesystem="/etc" with container because "/etc" is reserved by the container framework
pressure-vessel-wrap[39001]: W: Not sharing path --filesystem="/sbin" with container because "/sbin" is reserved by the container framework
Executing cd /usr/bin
------------------------------------------------------
warning: Unknown file arch of /home/ashton/.cache/protontricks/proton/Proton Hotfix/bin/wineserver.
------------------------------------------------------
------------------------------------------------------
warning: Unknown file arch of /home/ashton/.cache/protontricks/proton/Proton Hotfix/bin/wine.
------------------------------------------------------
------------------------------------------------------
warning: You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
------------------------------------------------------
warning: WoW64 type could not be detected.
------------------------------------------------------
Using winetricks 20250102 - sha256sum: 53194dead910f8a5eb1deacaa4773d4e48f5873633d18ab1ecd6fdb0cb92243b with wine-10.0 and WINEARCH=win64
Executing w_do_call d3dcompiler_47
------------------------------------------------------
warning: Unknown file arch of /home/ashton/.cache/protontricks/proton/Proton Hotfix/bin/wineserver.
------------------------------------------------------
------------------------------------------------------
warning: Unknown file arch of /home/ashton/.cache/protontricks/proton/Proton Hotfix/bin/wine.
------------------------------------------------------
------------------------------------------------------
warning: You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
------------------------------------------------------
warning: WoW64 type could not be detected.
------------------------------------------------------
d3dcompiler_47 already installed, skipping
```

no black screen, just fatal error on launch.

**Comment by simifor** [comment]
@ashtonx it's working normally on my end with proton 9/10/experimental, at least the normal launch, I don't have a VR setup to check the other option. You don't need to install anything manual, either.

Share your setup (hardware, driver versions, etc) and run the game after adding the following game launch parameter: `PROTON_LOG=1 %command%`. This will generate a file called `steam-895870.log` in your home folder, upload it (compress it if it happens to be too big for github).
